products/content/content-briefs/representatives.md

```diff
@@ -0,0 +1,118 @@
+# VA content brief for representatives
+
+The sitewide content team created this content brief. It's for teams who are designing forms or products that involve representatives acting on behalf of a claimant or applicant.
+
+**Have questions or feedback on this brief or want to add something?**
+- Send us a slack in #sitewide-content-ia.
+
+## Sections
+
+- [Overview](#overview)
+- [Related terminology](#related-terminology)
+- [Content considerations specific to sign in and identity verification](#content-considerations-specific-to-representatives)
+- [Organic SEO and VA.gov search](#organic-seo-and-va.gov-search)
+- [Content-specific research findings](#content-specific-research-findings)
+- [Content pages related to this topic](#content-pages-related-to-this-topic)
+- [Other notes](#other-notes)
+
+## Overview
+
+Speaking plainly, compassionately, and consistently across VA helps us build trust with Veterans, service members, and their family members. As we continue to move toward an omnichannel experience and multilingual content, consistency becomes even more important.  
+
+This brief will help your team create content for your product that aligns with the VA.gov voice and content standards and that uses consistent language about representatives--people signing and acting on behalf of Veterans, service members, and family members. It will also help us all track content decisions so we can all stay aligned from discovery through post-launch iterations. This is a living document. 
+
+[Read our top 10 plain language standards in the VA.gov content style guide for more](https://design.va.gov/content-style-guide/content-principles/plain-language)
+
+## Related terminology
+
+**accredited representative**
+
+We use this term to refer to a person or organization who's authorized to provide help with VA benefit claims to Veterans and service members, as well as their dependents and survivors. In order for an accredited representative to sign for someone else, the claimant or the representative must submit VA Form 21-22 or VA Form 21-22a. 
+
+Note on usage: Use "VA accredited representative" in the first instance. After that, "accredited representative" is OK. Lowercase, and define on first appropriate instance as “a person or organization who's authorized to help with VA benefit forms.” Include the definition in another sentence, as a parenthetical, or set off with commas. 
+
+**alternate signer**
+
+We use this term for a person who's certified to sign certain VA forms or applications for a claimant. A person can certify as an alternate signer by submitting VA Form 21-0972.
```

Comment:
"who's certified to sign certain VA forms on behalf of someone else. For example, signing a disability claim on behalf of a Veteran."

Reply:
I'm trying to keep "form" as the general term for the thing they're signing, and avoid making "claimant" the only term for the person if it might be broader than that. Does this work?

Reply:
Yes, that's a lot better. Thank you!!